Sources/PenguinStructures/Graphs/GraphProtocol.swift

```diff
@@ -93,28 +94,28 @@ public protocol VertexListGraph: GraphProtocol {
     /// Note: `vertexCount` might have O(V) complexity.
     var vertexCount: Int { get }
 
-    /// A collection of verticies.
+    /// The collection of vertex identifiers.
```

Comment:
```suggestion
    /// The identifiers of all vertices.
```
Given the name being documented, though, you might want to think about something like:
```suggestion
    /// All of the graph's vertices.
```